backend/routes/task.routes.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
+import { TaskController } from "../controllers/task.controller";
+import { authorize, UserType } from '../middleware/credentials.middleware';
+
+const router = express.Router();
+const taskController = new TaskController();
+
+router.get('/one',authorize([UserType.ADMIN]), taskController.getOne.bind(taskController))
+router.get('/oneOwned', authorize([UserType.MANAGER, UserType.ADMIN]), taskController.getOneOwned)
```

Comment:
It's generally recommended that endpoint paths be `kebab-cased` if they're comprised of multiple words, you'll also see `camelCased` paths, but commonly they're `kebab-case`, no need to change this now but just FYI